### PR TITLE
Docs migrations

### LINKED ISSUE #577: Migrate provider docs from website/**md.html into docs/**.md
We need to start slowly migrating docs from the current format into the new as described in https://www.terraform.io/docs/registry/providers/docs.html

### PR DESCRIPTION
**Pull request by lego963**
Resolve: #577

### CODE REVIEW

**Review by gtema — CHANGES_REQUESTED** [review]
Please go through the readme and fix links to old repo location (unfortunately GH does not allow commenting lines not touched by PR)

**Comment by gtema** [comment]
you should not remove the "--- frontmatter ---" headers from files. At least page_title should stay (actually perhaps start using subcategory). Read carefully https://www.terraform.io/docs/registry/providers/docs.html

**Comment by lego963** [comment]
Yes, I will return it, now I format the documents and fix them with strange or insufficient info

File: docs/data-sources/antiddos_v1.md [hunk 1]
```diff
@@ -1,63 +1,59 @@
...
-
-* `traffic_cleaning_status` - The traffic cleaning status.
-
-* `trigger_bps` - The traffic at the triggering point.
-
-* `trigger_pps` - The packet rate at the triggering point.
-
-* `trigger_http_pps` - The HTTP request rate at the triggering point.
+---
+subcategory: "AntiDDos"
```
Comment (outcatcher) [inline docs/data-sources/antiddos_v1.md]: It should be `DDoS`, not `DDos`

**Review by gtema — DISMISSED** [review]
Whew.
I don't know whether we will get problems with removal of `title`, but we can re-add it then.

**Review by outcatcher — CHANGES_REQUESTED** [review]

**Comment by lego963** [comment]
>`page_title - The title of this document, which will display in the docs navigation. This is only required for documents in the guides/ folder.`

according to https://www.terraform.io/docs/registry/providers/docs.html

**Comment by gtema** [comment]
> > `page_title - The title of this document, which will display in the docs navigation. This is only required for documents in the guides/ folder.`
> 
> according to https://www.terraform.io/docs/registry/providers/docs.html

it's not about "requires", it's about how the page name would be shown by the render tool, but even before it was not very much useful. On the other hand all other providers do prefix pages with the provider name (like it it currently)

**Comment by gtema** [comment]
hmm, the build is also failing in a weird way

**Review by outcatcher — DISMISSED** [review]

**Review by gtema — APPROVED** [review]
No clue how to review 271 files other that simply by diagonal

**Pull request merged**